source/configuration_manual/sql.rst

```diff
@@ -0,0 +1,526 @@
+===========
+SQL drivers
+===========
+
+Usage
+=====
+
+Dovecot supports using SQL drivers in :ref:`dictionaries <dict>` and
+:ref:`authentication <authentication-sql>`.
+
+To use a particular driver, you use
+
+.. code::
+
+   driver = <driver name>
+   connect_string = <driver specific options>
+
+in your dictionary or authentication configuration file.
+The connect_string is a space separated list of ``key=value`` pairs.
+
+Cassandra
+=========
+
```

Comment:
I think these would be better split to their own pages. Now this Cassandra stuff takes almost the whole page.